Add a checkbox to configure the aligned mode.

knz opened the pull request:
sqlfmt code to accompany https://github.com/cockroachdb/cockroach/pull/27260

First commit from #18.

knz commented:
I also added a commit in this PR to add some mutation statements to the example.

mjibson commented:
I did a big frontend refactor and messed this PR all up. I'm happy to fix it myself if you'd rather not deal with it.

knz commented:
I fixed it. No worries.

Note that I reverted your investion of the checkbox "spaces". If you want to swap its meaning, please then don't do this in javascript and instead do it server-side (and change the URL key used for it). This way the form still works when javascript is disabled.
